object-storage/index.js

```diff
@@ -54,7 +54,7 @@ async function store(bucket, key, data, compress = false) {
       Bucket: bucket,
       Key: key,
       Body: passThroughStream,
-      ACL: "public-read",
+      ACL: config.aclPermissions,
```

Comment:
Can you please rename this to `config.storageACL` to be consistent with the other config settings.

Reply:
renamed it!